int32 camera_active
---
int32 ack
